### PR TITLE
📝 Reduce usage of shields.io badges so they render correctly

### PR DESCRIPTION
**Pull request by madchutney**
### Description

These badges for PyPI rarely load using `shields.io` so instead use a different badge from `fury.io` for the version and remove all other badges.

### Test Coverage

- [ ]  This change is covered by existing or additional automated tests.
- [x]  Manual testing has been performed (and evidence provided) as automated testing was not feasible.
- [ ]  Additional tests are not required for this change (e.g. documentation update).

### CODE REVIEW

**Comment by acabarbaye** [comment]
Could https://badgen.net/ be used instead of shields.io to generate all the missing badges?
I would personally prefer the name of the project being the master header of the readme rather than the package name.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Mbed Project
+# Package: Mbed Project
```
Comment (acabarbaye) [inline README.md:1]: I would remove Package from the title

**Comment by acabarbaye** [comment]
Looks great!

File: README.md (hunk 1 above)
Comment (madchutney) [reply README.md]: Done

**Pull request merged**